packages/yourloops/components/patient-data/patient-data.hook.ts

```diff
@@ -257,7 +275,6 @@ export const usePatientData = (): usePatientDataResult => {
       })
       setDataUtil(dataUtil)
     }
-    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❔ question:** ‏Can you explain this change to me?